examples/demo-stf/README.md

```diff
@@ -33,90 +33,90 @@ several parameters that specify its exact behavior. In order, these four generic
 1. `Context`: a per-transaction struct containing the message's sender. This also provides specs for storage access, so we use different `Context`
    implementations for Native and ZK execution. In ZK, we read values non-deterministically from hints and check them against a merkle tree, while in
    native mode we just read values straight from disk.
-2. `TxVerifier`: a struct that verifies the signatures on transactions and deserializes them into messages
-3. `Runtime`: a collection of modules which make up the rollup's public interface
-4. `TxHooks`: a set of functions which are invoked at various points in the transaction lifecycle
+2. `Runtime`: a collection of modules which make up the rollup's public interface
+3. `TxHooks & ApplyBlobHooks` derived from the `Runtime`: a set of functions which are invoked at various points in the transaction lifecycle.
```

Comment:
Since these generics don't show up on the AppTemplate anymore, I think we can drop them from the documentation.  We should also change the line above this list from "In order, these four generics are:" to "In order, these generics are:"

Reply:
Fixed